### ISSUE TITLE
sort by alphabetical  order

### ISSUE THREAD

**Issue by yiqiquhuxi** [balderdashy/sails#2475]
I can't  sort by first alphabetical order .Only by method of the query. Should I use 

the sails-self method?
![qq 20141208171952](https://cloud.githubusercontent.com/assets/7865661/5337050/10fbce20-7eff-11e4-8914-5c49ffb627a2.jpg)

**Comment by CWyrtzen** [comment]
Is this still relevant with the latest releases?

**Comment by calidion** [comment]
when there is no method provide by order method. this problem won't be solved.

**Comment by sailsbot** [comment]
Thanks for posting, @yiqiquhuxi.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
